pkgs/os-specific/linux/systemd/bootchart.nix

```diff
@@ -21,6 +21,13 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ systemd ];
 
+  patches = [
+    # https://github.com/systemd/systemd-bootchart/pull/10
+    ./bootchart-fix-processes-graph.patch
```

Comment:
It's better to use `fetchpatch` there -- we try not to store any patches in the tree. For example, the first patch's URL is https://github.com/akavel/systemd-bootchart/commit/c5e0f0e2669b048502fdd9c4d1116bf2fb22c20f.patch